apps/landingPage/src/components/Headline.js

```diff
@@ -13,87 +13,137 @@ import airport from '../images/airport.jpg';
+                </Heading>
               </Text>
-            </WrapperText>
-            <WrapperButtons>
-              <Button>
-                <LinkNoStyle
-                  href={margarita}
-                  color={defaultTokens.paletteWhite}
-                >
-                  Get started
-                </LinkNoStyle>
-              </Button>
-              <WrapperRightButton>
-                <Button type="white">
+            </HeaderAndDesignWrapper>
+            <ContainerIcons>
+              <img src={kiwiLogo} height={43} alt="kiwi_logo" />
+              <img src={reactLogo} height={45} alt="react_logo" />
+              <img src={tequilaLogo} height={43} alt="tequila_logo" />
+              <img src={graphqlLogo} height={43} alt="graphql_logo" />
+            </ContainerIcons>
+          </ContainerTop>
+          {isTablet && (
+            <>
+              <MobileGradientAccent>
+                <PlaneImageMobile />
+              </MobileGradientAccent>
+            </>
+          )}
+          <ContainerBottom>
+            <ContainerLeftText>
+              <Heading>Get started</Heading>
+              <WrapperText>
+                <Text size="large">
+                  PLACEHOLDER!!!!!You've had a look at Tequila, but you've
+                  preffered yourTequila Tequila mixed with something else. Just
+                  grab Margarita.You've had a look at Tequila, but you've always
+                  preffered your Tequila mixed with something else.
+                </Text>
+              </WrapperText>
+              <WrapperButtons>
+                <Button block={isTablet || false} size={buttonSize}>
                   <LinkNoStyle
-                    href={github}
-                    color={defaultTokens.colorTextButtonSecondary}
+                    href={margarita}
+                    color={defaultTokens.paletteWhite}
                   >
-                    Link to Github
+                    Get started
                   </LinkNoStyle>
                 </Button>
-              </WrapperRightButton>
-            </WrapperButtons>
-          </ContainerLeftText>
-          <PlaneImage />
-        </ContainerBottom>
-      </Container>
-    </HeadlineContainer>
-  );
+                <WrapperRightButton>
+                  <Button
+                    type="white"
+                    block={isTablet || false}
+                    size={buttonSize}
+                  >
+                    <LinkNoStyle
+                      href={github}
+                      color={defaultTokens.colorTextButtonSecondary}
+                    >
+                      Link to Github
+                    </LinkNoStyle>
+                  </Button>
+                </WrapperRightButton>
+              </WrapperButtons>
+            </ContainerLeftText>
+            <PlaneImage displayMobile="none" />
+          </ContainerBottom>
+        </Container>
+      </HeadlineContainer>
+    );
+  }
```

Comment:
Also this component is growing quite big, would it be possible to separate it to smaller logical components?

Reply:
Extracting *Get Started* part to the separate component